database/migrations/2021_03_06_100000_add_regions_to_zip_codes_table.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+use Tipoff\Addresses\Models\Region;
+
+class AddRegionsToZipCodesTable extends Migration
+{
+    public function up()
+    {
+        Schema::table('zip_codes', function (Blueprint $table) {
+               $table->foreignIdFor(Region::class)->nullable();
```

Comment:
This package isn't in production yet, so I'm going to add the field directly to the zip_codes migration.